build.gradle.kts

```diff
@@ -197,8 +197,7 @@ tasks.register("javaPostCommit") {
   dependsOn(":sdks:java:extensions:google-cloud-platform-core:postCommit")
   dependsOn(":sdks:java:extensions:zetasketch:postCommit")
   dependsOn(":sdks:java:io:google-cloud-platform:postCommit")
-  dependsOn(":sdks:java:io:amazon-web-services:integrationTest")
-  dependsOn(":sdks:java:io:amazon-web-services2:integrationTest")
+  dependsOn(":sdks:java:io:kinesis:integrationTest")
```

Comment:
Why is this added?

Reply:
I think is related to a merge conflict with other branch that is working in the tests. Let me talk about it with the team and do the changes.